### ISSUE TITLE
New Big Picture: `PartnerEventOverlayContainer` fails to be hidden

### ISSUE THREAD

**Issue by ticky** [ValveSoftware/steam-for-linux#9018]
#### Your system information

* Steam client version (build number or date): `1671254119`
* Distribution (e.g. Ubuntu): SteamOS 3.3.3
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
I expect that the UI should not end up in an unusable state through normal use. I don't have any unusual configuration, and this bug has been with me since the day I first got my Steam Deck.

After opening and closing any game update, either from the "Home" page, or from an individual game's library, the background blur element persists, so the entire UI remains blurred until you either reboot, or use the "switch to desktop" button from the power menu, at which point you can back out and the blur will be removed.

I've had a peek in there with the CEF remote debugger, and it appears it is failing to hide this element after you close the news item:

```html
<div class="basiclibrary_TransitionWrapper_1_fls basiclibrary_OverlayAppDetailsTransitionEntered_3pmme"></div>
```

I haven't been able to peek into any JavaScript, so this is about as far as I can get.

For me this happens every single time I open any news article. It doesn't happen on my friend's Steam Deck. There are no obvious configuration differences between the two, and they were delivered a small number of months apart.

#### Steps for reproducing this issue:

1. Have a Steam Deck
2. Open "What's New" section
    ![20221218220701_1](https://user-images.githubusercontent.com/282113/208359029-f7a0936a-87e9-47d3-acc2-919bb4624ef2.jpg)
4. Click any news item
   ![20221218220711_1](https://user-images.githubusercontent.com/282113/208359060-52731a6e-e082-4833-bdd0-e658408c2d98.jpg)
5. Close out of the news item
7. Observe that the UI is now blurred
   ![20221218220714_1](https://user-images.githubusercontent.com/282113/208359109-dc3cbc3d-9d73-4a20-a35b-0d26ae08c66c.jpg)

**Comment by kisak-valve** [comment]
Hello @ticky, in a quick test I was unable to reproduce this issue. It's likely there's an additional factor that's needed to reproduce this issue.

**Comment by ticky** [comment]
Fair enough, are you able to provide any insight into what OverlayAppDetails might be doing here? Any guidance on what further information to gather?

**Comment by smcv** [comment]
I've seen this happen, but not reproducibly - there might be a race condition or some other intermittent issue?

**Comment by TrainDoctor** [comment]
I have had this exact issue since day one of having my deck and doing everything but fully wiping my deck (a steamos reinstall but preserving the home partition). I can produce this issue at any time and I can collect whatever logs are requested of me.

**Comment by ticky** [comment]
I know of at least one other person seeing this same bug on the newly-released desktop new big picture https://cloudisland.nz/@PetraOleum/109799048181945064

**Comment by ticky** [comment]
This got me digging in again and I believe I have discovered the additional factor: _Low Performance Mode_.

---

The code for the `PartnerEventOverlayContainer` (in `steamloopback.host/sp.js?chunkhash=3caaf6adbbbccd5f08bc`) has a reference to `BLowPerfMode`, the return value of which is then passed to the parent of something with a `className` of `TransitionWrapper` as the properties `appear`, `enter` and `exit`. This tipped me off to test it, and _this_ is the bit that's different; I always run Steam in low bandwidth/low performance mode (because I find the performance in the usual mode with all the images and blur effects too slow, even on beefy hardware), and I did so with my Steam Deck too.

Turning off "Low Performance Mode" in Desktop Steam's settings (I'm not sure the setting is exposed in new big picture/the normal Steam Deck UI) suddenly makes the blurred background disappear as it should on my Steam Deck. Likewise, turning it back on reproduces the issue.

I suspect this is enough information for internal Valve folks to find the actual underlying bug (presumably with zero-duration exit transitions?) and patch it up.

**Comment by TrainDoctor** [comment]
@ticky thanks for testing this, I've been far too busy with Decky. I was just able to reproduce the issue and then unable to reproduce it once I disabled the low performance mode.
